src/main/java/no/sikt/nva/pubchannels/channelregistrycache/LevelForYear.java

```diff
@@ -1,5 +1,38 @@
 package no.sikt.nva.pubchannels.channelregistrycache;
 
-public record LevelForYear(String year, String level) {
+import nva.commons.core.JacocoGenerated;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbAttribute;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbImmutable;
 
+@DynamoDbImmutable(builder = LevelForYear.Builder.class)
+public record LevelForYear(@DynamoDbAttribute("year") String year, @DynamoDbAttribute("level") String level) {
+
+    @JacocoGenerated
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    @JacocoGenerated
```

Comment:
Why does this need to be annotated? Is it not used in tests?

Reply:
Nope, is needed for dynamo db entry serialization